styles.css

```diff
@@ -84,10 +83,400 @@
 }
 
 .socialmediaicons {
+  font-size: 20px;
   list-style-type: none;
   display: flex;
   flex-direction: row;
   align-items: center;
   padding: 0;
   place-content: space-between;
 }
+
+.work-section {
+  margin: 0;
+  background-color: var(--white-color);
+  padding-bottom: 120px;
+}
+
+.RWbox {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 120px 0;
+}
+
+.RWork {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 0;
+  gap: 24px;
+  width: 327px;
+  height: 107px;
+  height: 83px;
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 30px;
+  line-height: 52px;
+}
+
+.workline {
+  width: 327px;
+  border: 1px solid #212121;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.workone {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  padding: var(--padding-work);
+  height: 568px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.workphoto {
+  padding: var(--padding-work);
+  padding-top: 30px;
+}
+
+.first-Rwork {
+  padding: var(--padding-work);
+  margin: 20px 0;
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  color: #091e42;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.work-dicreption {
+  padding: var(--padding-work);
+  justify-content: center;
+  margin-top: 0;
+  margin-bottom: 1em;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.languages-Fwork {
+  display: flex;
+  margin-bottom: 1em;
+  padding: var(--padding-work);
+  order: 2;
+  flex-grow: 0;
+}
+
+.see-project {
+  padding: var(--padding-work);
+  flex: none;
+  order: 3;
+  flex-grow: 0;
+  width: 124px;
+  height: 48px;
+}
+
+.diffrent-work {
+  display: grid;
+  grid-template-rows: 6fr 6fr 6fr 6fr 6fr 6fr;
+  justify-content: space-evenly;
+  row-gap: 50px;
+  padding-top: 20px;
+  padding-left: 20px;
+  padding-right: 20px;
+  margin-top: 40px;
+  margin-bottom: 0;
+}
+
+.worktwo {
+  display: flex;
+  flex-direction: column;
+
+  /* align-items: center; */
+  height: 350px;
+  border-radius: 0;
+  background: url(/img/Img\ Placeholder1.svg);
```

Comment:
- [ ] Kindly fix this link it's broken and not showing the image in the webpage. I don't think you can make spaces in the names of the images. A way to fix this would be to wrap the link in quotations (" ") and rename the image example: Img_Placeholder1 👍